### PR TITLE
added `pipe` package

### PR DESCRIPTION
**Pull request by TClark1011**
NOTE: I did not add the rollup config because I wasn't sure how I should configure it

### CODE REVIEW

File: md-variables.json [hunk 1]
```diff
@@ -1599,5 +1599,16 @@
       "const squareFn = (number) => number * number;",
       "deepMapValues({ a: 1, b: { c: 2, d: { e: 3 } } }, squareFn); // => { a: 1, b: { c: 4, d: { e: 9 } } }"
     ]
+  },
+  "just-flow": {
```
Comment (angus-c) [inline md-variables.json:1603]: Rename `just-flow` to `just-pipe`

File: test/function-pipe/index.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  }
+  function reverse(arr) {
+    return [...arr].reverse();
+  }
+
+  t.equal(pipe([1, 2, 3], prependZero, reverse, last), 0);
+  t.end();
+});
+
+test('Throws if only give 0 or 1 parameters', function(t) {
```
Comment (angus-c) [inline test/function-pipe/index.js:29]: can you add the fail case where second param is a non-function

File: test/function-pipe/index.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  t.plan(3);
+  t.equal(pipe('hello world', camelCase), 'helloWorld');
+
+  function addOne(a) {
+    return a + 1;
+  }
+  function double(b) {
+    return b * 2;
+  }
+  t.equal(pipe(3, addOne, double), 8);
```
Comment (angus-c) [inline test/function-pipe/index.js:16]: Would also be good to tests with args 2 and 3 swapped

**Review by angus-c — CHANGES_REQUESTED** [review]
Thanks for this! Just a few small issues. Also did you run `yarn generate-readmes`? I would have expected updates to global README.ms too.

**Comment by TClark1011** [comment]
@angus-c requested changes have been made, sorry for the delay

**Comment by angus-c** [comment]
Sorry for the even bigger delay from me! Thanks for making the changes. I still don't see a global README for some reason. I'm going to merge now anyway and try running the script myself—and the one for `.mjs` generation. Might not get to doing those updates it immediately, so it won't be published right away.
Thanks again!

**Comment by angus-c** [comment]
Sorry for the even bigger delay from me! Thanks for making the changes. I still don't see a global README for some reason. I'm going to merge now anyway and try running the script myself—and the one for `.mjs` generation. Might not get to doing those updates it immediately, so it won't be published right away.
Thanks again!

**Comment by angus-c** [comment]
Ok my bad it was there but not in alpha order so I didn't see in global README
Re-ordered https://github.com/angus-c/just/pull/529

**Comment by angus-c** [comment]
Published as just-pipe@1.0.0

**Comment by angus-c** [comment]
And now available in REPL
https://anguscroll.com/just/just-pipe

**Pull request merged**